addons/web/static/tests/views/list_tests.js

```diff
@@ -9059,6 +9059,39 @@ QUnit.module('Views', {
         list.destroy();
     });
 
+    QUnit.test('control panel buttons in multi editable grouped list views', async function (assert) {
+        assert.expect(5);
+
+        var list = await createView({
```

Comment:
you can now use `const` everywhere you can, and `let` when you can't, instead of `var`